lib/server/server.js

```diff
@@ -97,7 +97,11 @@ const serviceDependencies = Object.keys(servicePackage.dependencies)
 const serviceLockDependencies = servicePackageLock.dependencies
 
 if (cleanupPackage) {
-  execSync(cleanupPackage)
+  try {
+    execSync(cleanupPackage)
+  } catch (e) {
+    //
```

Comment:
Why the empty catch block?  If the command fails, it'll take a while for us to find out?

Reply:
Fair point